src/ethereum/frontier/spec.py

```diff
@@ -637,14 +717,20 @@ def recover_sender(tx: Transaction) -> Address:
     """
     Extracts the sender address from a transaction.
 
+    The v, r, and s values are the three components that make up the
+    signature of a transaction. In order to recover the sender of a
+    transaction the two components needed are the signature and the signing
```

Comment:
Since we have two sets of components here (components of the signature, and components needed to recover the sender), I think an example would be wise:

```suggestion
    transaction the two components needed are the signature (`v`, `r`, and `s`) and the signing
```